lxqtleave/translations/lxqt-hibernate_it.desktop

```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Name=Hibernate
+GenericName=Hibernate
+Comment=hibernate the machine
+Exec=lxqt-leave --hibernate
+Icon=system-suspend-hibernate
+Categories=LXQt;X-Leave
+OnlyShowIn=LXQt
+
+#TRANSLATIONS_DIR=../translations
+
+# Translations
```

Comment:
Please, remove all universal keys and leave here only the translated ones (\[it\]).